src/web/experiments/tests/newsletter-merch-unit-test.ts

```diff
@@ -0,0 +1,38 @@
+import { ABTest } from '@guardian/ab-core';
+
+export const newsletterMerchUnitLighthouse: ABTest = {
+    id: 'NewsletterMerchUnitLighthouse',
+    start: '2020-11-11',
+    expiry: '2020-12-01',
+    author: 'Josh Buckland & Alex Dufournet',
+    description: 'Show a newsletter advert in the merchandising unit to 50% of users',
+    audience: 1.0, // 0.01%
+    audienceOffset: 0.0,
+    successMeasure: 'We see increased engagement from users shown the Newsletters ad unit',
+    audienceCriteria: 'Website users only.',
+    idealOutcome: 'Investigate lighthouse segment engagement via newsletters',
+    showForSensitive: false,
+    canRun: () => true,
+    variants: [
+        {
+            id: 'control',
+            test: (): void => {},
+            impression: (impression) => {
+                impression();
+            },
+            success: (success) => {
+                success();
+            },
```

Comment:
This will immediately fire impression and success for your variants. 

Normally how you use this is by setting up a listener to call the callback like this:

```
 success: (success) => {
                // https://developer.mozilla.org/en-US/docs/Web/Guide/Events/Creating_and_triggering_events
                window.addEventListener('myTestSuccess', (e) => {
					success()                
				});

            },
```

Which you can then fire as a custom event like `window.dispatchEvent(new CustomEvent('myTestSuccess')`

Reply:
I'm not sure we need success and impression to fire in this case. The test is being used to pass key-values to ad manager which will then use it for targeting. If memory serves, we should be getting all impressions and success/conversion data from the existing tracking around the merch unit

Reply:
So given this is going to be handled by ad-manager, should I leave the implementation empty?
```
            impression: (impression) => {},
```